recipes/fastobo/meta.yaml

```diff
@@ -23,14 +23,17 @@ requirements:
     - clang_osx-64  # [osx]
     - clangxx_osx-64  # [osx]
     - pkg-config  # [osx]
+    - sysroot_linux-64 =2.17 # [linux and x86_64]
```

Comment:
This fixes error `library/std/src/sys/pal/unix/stack_overflow.rs:237: undefined reference to `getauxval'`